packages/sitecore-jss-nextjs/src/edge/personalize-middleware.ts

```diff
@@ -125,14 +133,19 @@ export class PersonalizeMiddleware {
     const language = req.nextUrl.locale || req.nextUrl.defaultLocale || 'en';
     let browserId = this.getBrowserId(req);
 
+    // Response will be provided if other middleware is run before us (e.g. redirects)
+    let response = res || NextResponse.next();
+
+    if (this.config.disabled && this.config.disabled(req, response)){
+      debug.personalize('skipped: personalize middleware is disabled');
```

Comment:
Use consistent 'skipped' debug message format
```suggestion
      debug.personalize('skipped (personalize middleware is disabled)');
```